features/step_definitions/registry.rb

```diff
@@ -71,9 +71,9 @@
   end
   begin
     step %Q/I switch to cluster admin pseudo user/
-    step %Q/I use the "default" project/
+    step %Q/I use the "openshift-image-registry" project/
```

Comment:
The project name change works for all 4.x versions ?

Reply:
For now, it's yes

Reply:
ack